### PR TITLE
SupergroupEntity Icons Spalte & Indizes

### PR DESCRIPTION
**Pull request by JonahKr**
## Hinzugefügt
- VocableInstrumentedTest.java

## Beschreibung
- Indizes hinzugefügt (Keine `trigger full table scans` warnings mehr)
- `String icon` Spalte in #10 
- InstrumentedTest für `Vocable`

## Tasks
- #15 

## Checkliste
- [x] Code auf **Bugs** getestet
- [x] Code auf [**Konventionen**](https://github.com/atalantus/Wokabel-App/wiki/Dokumentation-%7C-Code-Konventionen) getestet
- [x] Code im **[Wiki](https://github.com/atalantus/Wokabel-App/wiki) dokumentiert**

### CODE REVIEW

**Comment by atalantus** [comment]
So jetzt bitte einmal ordentlich
In den Titel kommt die super kurz beschreibung was man gemacht hat.

File: App/app/src/main/java/com/wokabel/app/wokabel/services/room/DatabaseAdapter.java [hunk 1]
```diff
@@ -54,16 +54,19 @@ Supergroup getSupergroupbyId(String id){
         return subdao.getSubgroupsbySupergroup(id);
     }
 
-    void insertVocable(Vocable vocable){
+    Vocable insertVocable(Vocable vocable){
```
Comment (atalantus) [inline App/app/src/main/java/com/wokabel/app/wokabel/services/room/DatabaseAdapter.java]: Wieso gibt er da das Objekt wieder zurück?

**Comment by atalantus** [comment]
## App startet nicht mehr!
    07-13 23:53:05.404 9728-9758/? E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #2
    Process: com.wokabel.app.wokabel, PID: 9728
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:353)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: java.lang.IllegalStateException: Room cannot verify the data integrity. Looks like you've changed schema but forgot to update the version number. You can simply fix this by increasing the version number.
        at android.arch.persistence.room.RoomOpenHelper.checkIdentity(RoomOpenHelper.java:135)
        at android.arch.persistence.room.RoomOpenHelper.onOpen(RoomOpenHelper.java:115)
        at android.arch.persistence.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onOpen(FrameworkSQLiteOpenHelper.java:151)
        at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:349)
        at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:238)
        at android.arch.persistence.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableSupportDatabase(FrameworkSQLiteOpenHelper.java:96)
        at android.arch.persistence.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.java:54)
        at android.arch.persistence.room.RoomDatabase.compileStatement(RoomDatabase.java:244)
        at android.arch.persistence.room.SharedSQLiteStatement.createNewStatement(SharedSQLiteStatement.java:65)
        at android.arch.persistence.room.SharedSQLiteStatement.getStmt(SharedSQLiteStatement.java:72)
        at android.arch.persistence.room.SharedSQLiteStatement.acquire(SharedSQLiteStatement.java:87)
        at com.wokabel.app.wokabel.services.room.VocableDao_Impl.deleteAll(VocableDao_Impl.java:98)
        at com.wokabel.app.wokabel.services.room.DatabaseAdapter.deleteDatabaseContent(DatabaseAdapter.java:77)
        at com.wokabel.app.wokabel.services.room.DatabaseAdapter$LoadData.doInBackground(DatabaseAdapter.java:129)
        at com.wokabel.app.wokabel.services.room.DatabaseAdapter$LoadData.doInBackground(DatabaseAdapter.java:118)
        at android.os.AsyncTask$2.call(AsyncTask.java:333)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636) 
        at java.lang.Thread.run(Thread.java:764)

**Comment by JonahKr** [comment]
👍

**Pull request merged**